### ISSUE TITLE
Image & bullet points not rendering?

### ISSUE THREAD

**Issue by erich5168** [ckRobinson/multi-column-markdown#34]
Hi, thank you so much creating this plugin!  Its a greate alternative.

Obsidian: 0.15.9
multi-column : 0.7.2

I thought I created an issue way back but I guess I never clicked "Submit".  These problems happend in version 0.6.x, so I think its safe to rule out that this is a 0.7.2 problem.  Maybe its more like me doing it all wrong!  If could take a look it would be greately appreciated.

1. Image not rendering.  This is what I have on my notes

=== multi-column-start: ID_2a6p
```column-settings
Number of Columns: 2
Largest Column: standard
border: off
```

![[Pasted image 20220724182242.png]]

--- end-column ---

###### Test2![[Pasted image 20220724182525.png]]

=== multi-column-end

**Editing:** I see it render in editing mode... so the link should be pointing to the right places.
![Screen Shot 2022-08-24 at 12 12 22 PM](https://user-images.githubusercontent.com/31515782/186327276-aaf67564-4e2f-494d-a656-22347117d510.png)

**Read mode:** Images not showing up
![Screen Shot 2022-08-24 at 12 04 50 PM](https://user-images.githubusercontent.com/31515782/186326405-f82cb16b-c8a3-4851-a33b-4d953288b1ef.png)

2. Bullet points
=== multi-column-start: ID_dams
```column-settings
Number of Columns: 2
Largest Column: standard
```
###### Lithium batteries 3000 amp
* 1~1.5 fully charge
* 60W charing would fry the battery
* 600 Charge cycles
* Heat issue
=== end-column ===
###### Real Graphene Batteries
20min full charge
Safe to use 60W charger
1500 Charge cycles
Runes much cooler and generates much less heat.
=== multi-column-end

**Read mode:** Bullets not showing
![Screen Shot 2022-08-24 at 12 07 28 PM](https://user-images.githubusercontent.com/31515782/186326665-9d22b7a0-b919-4c9e-9121-e0e1d1a83366.png)

Thanks so much.

**Comment by ckRobinson** [comment]
Hi there!

1.
For the first bug, I copied and pasted your markdown into Obsidian on both of my machines and I don't see the issue on my end.

<img width="871" alt="Screen Shot 2022-08-30 at 11 32 13 AM" src="https://user-images.githubusercontent.com/70980846/187518156-1f4db2d4-7d3e-4e6e-a52d-80c1d4803f39.png">
<img width="884" alt="Screen Shot 2022-08-30 at 11 32 33 AM" src="https://user-images.githubusercontent.com/70980846/187518184-69e70b4c-2757-4f36-95ca-7c70426170ba.png">

I had to replace your image links with screenshots taken from your post, but it renders properly so I am not sure what the issue would be. I also have used images in columns many times in my own daily use and have not seen this issue. Maybe there is a conflict with another plugin that is causing the problem? Really at a loss but will continue to investigate.

2. 
Again copied the markdown in and this one does not render like you see:

<img width="874" alt="Screen Shot 2022-08-30 at 11 35 24 AM" src="https://user-images.githubusercontent.com/70980846/187518967-e0157035-45c8-41f7-a197-a89671c5f9aa.png">
<img width="418" alt="Screen Shot 2022-08-30 at 11 34 46 AM" src="https://user-images.githubusercontent.com/70980846/187518958-0ed9c009-70a6-4106-90d4-3a7d5dfc5a1d.png">

However, at least in the text you used in the post, the syntax is missing newlines around the bullets and the column break tag:
<img width="380" alt="Screen Shot 2022-08-30 at 11 35 51 AM" src="https://user-images.githubusercontent.com/70980846/187519151-eeb81405-b7d0-494a-a23f-10c64c0eee52.png">
<img width="880" alt="Screen Shot 2022-08-30 at 11 36 04 AM" src="https://user-images.githubusercontent.com/70980846/187519159-f484147a-67c4-4e5b-ada7-967ce8f1e1aa.png">

The "requirement" to surround bullet lists (and other items) with newlines is built into how obsidian sends data to the plugins. I have an update in the works that may help with this problem but it is a major rework and it will take some time.

**Comment by erich5168** [comment]
Thank you so much for taking the time to look into this.  

1. Just FYI, my community plugins are:
- Admonition(call backs)
- Dataview
- Find unlinked files and unresolved links
- Highlightr
- LanguageToolIntegration
- Multi-Column Markdown
- Past URL into selection
- Recent files

2. Got it, I was missing new lines surounding bullet list.

Thanks again for your help.

**Comment by ckRobinson** [comment]
I have pushed a fix in 0.7.3 that should fix this on your end. If the problem persists let me know and I will re-open the issue.
